samples/modules/lvgl/src/main.c

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright (c) 2023 Fabian Blatz <fabianblatz@gmail.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <zephyr/kernel.h>
+#include <zephyr/drivers/display.h>
+#include <lvgl.h>
+#include <lv_demos.h>
+
+#define LOG_LEVEL CONFIG_LOG_DEFAULT_LEVEL
+#include <zephyr/logging/log.h>
+LOG_MODULE_REGISTER(app);
+
+int main(void)
+{
+	const struct device *display_dev;
+
+	display_dev = DEVICE_DT_GET(DT_CHOSEN(zephyr_display));
+	if (!device_is_ready(display_dev)) {
+		LOG_ERR("Device not ready, aborting test");
+		return 0;
+	}
+
+#if defined(CONFIG_LV_USE_DEMO_MUSIC)
+	lv_demo_music();
+#elif defined(CONFIG_LV_USE_DEMO_BENCHMARK)
+	lv_demo_benchmark();
+#elif defined(CONFIG_LV_USE_DEMO_STRESS)
+	lv_demo_stress();
+#else
+	BUILD_ASSERT(0, "Enable one of the demos CONFIG_LV_USE_DEMO_MUSIC, "
+			"CONFIG_LV_USE_DEMO_BENCHMARK, CONFIG_LV_USE_DEMO_STRESS");
```

Comment:
I would suggest adding a `Kconfig` to the sample with a `choice` option for each demo available.

Nit: you can use the `#error` preprocessor:
```suggestion
#error Enable one of the demos CONFIG_LV_USE_DEMO_MUSIC, CONFIG_LV_USE_DEMO_BENCHMARK \
       or CONFIG_LV_USE_DEMO_STRESS.
```

Reply:
Not too sure how to gracefully handle the Kconfig symbols, I can not reuse the same symbols because they are defined outside of the prompt within the modules Kconfig.

Reply:
You can define specific symbols for the sample and `select` the corresponding module symbol.

Reply:
yeah thats what I end up doing :+1: